src/Sylius/Behat/Context/Setup/TaxContext.php

```diff
@@ -0,0 +1,125 @@
+
+namespace Sylius\Behat\Context\Setup;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Addressing\Model\ZoneInterface;
+use Sylius\Component\Core\Model\TaxRateInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class TaxContext implements Context
+{
+    /**
+     * @var FactoryInterface
+     */
+    private $taxRateFactory;
+
+    /**
+     * @var FactoryInterface
+     */
+    private $taxCategoryFactory;
+
+    /**
+     * @var RepositoryInterface
+     */
+    private $taxRateRepository;
+
+    /**
+     * @var RepositoryInterface
+     */
+    private $taxCategoryRepository;
+
+    /**
+     * @var RepositoryInterface
+     */
+    private $zoneRepository;
+
+    /**
+     * @param FactoryInterface $taxRateFactory
+     * @param FactoryInterface $taxCategoryFactory
+     * @param RepositoryInterface $taxRateRepository
+     * @param RepositoryInterface $taxCategoryRepository
+     * @param RepositoryInterface $zoneRepository
+     */
+    public function __construct(
+        FactoryInterface $taxRateFactory,
+        FactoryInterface $taxCategoryFactory,
+        RepositoryInterface $taxRateRepository,
+        RepositoryInterface $taxCategoryRepository,
+        RepositoryInterface $zoneRepository
+    ) {
+        $this->taxRateFactory = $taxRateFactory;
+        $this->taxCategoryFactory = $taxCategoryFactory;
+        $this->taxRateRepository = $taxRateRepository;
+        $this->taxCategoryRepository = $taxCategoryRepository;
+        $this->zoneRepository = $zoneRepository;
+    }
+
+    /**
+     * @Transform /^"([^"]+)" tax category$/
+     * @Transform /^tax category "([^"]+)"$/
+     */
+    public function castTaxCategoryNameToTaxCategory($taxCategoryName)
+    {
+        if (null === $taxCategory = $this->taxCategoryRepository->findOneBy(array('name' => $taxCategoryName))) {
+            throw new \Exception('Tax category with name "'.$taxCategoryName.'" does not exist');
+        }
+
+        return $taxCategory;
+    }
+
+    /**
+     * @Given /^store has "([^"]+)" tax rate of ([^"]+)% for "([^"]+)" within ("([^"]+)" zone)$/
+     */
+    public function storeHasTaxRateWithinZone($taxRateName, $taxRateAmount, $taxCategoryName, ZoneInterface $taxZone)
+    {
+        $taxCategory = $this->taxCategoryFactory->createNew();
+        $taxCategory->setName($taxCategoryName);
+        $taxCategory->setCode($this->getCodeFromName($taxCategoryName));
+
+        $this->taxCategoryRepository->add($taxCategory);
+
+        $taxRate = $this->taxRateFactory->createNew();
+        $taxRate->setName($taxRateName);
+        $taxRate->setCode($this->getCodeFromName($taxRateName));
+        $taxRate->setZone($taxZone);
+        $taxRate->setAmount($this->getAmountFromString($taxRateAmount));
+        $taxRate->setCategory($taxCategory);
+        $taxRate->setCalculator('default');
```

Comment:
Sadly, there are no constants for tax rate calculators, at least I couldn't find any :(